sdk/storage/azure-storage-blob/src/test/java/com/azure/android/storage/blob/BlobTest.java

```diff
@@ -0,0 +1,443 @@
+        List<String> blockIds = new ArrayList<>();
+        blockIds.add(blockId);
+        syncClient.commitBlockList(containerName, blobName, blockIds, false);
+    }
+
+    @After
+    public void teardownTest() {
+        syncClient.deleteContainer(containerName);
+    }
+
+    @Test
+    public void getPropertiesValues() {
+        // When
+        BlobGetPropertiesHeaders response = syncClient.getBlobProperties(containerName, blobName);
+
+        // Then
+        assertNotNull(response.getETag());
+//        assertFalse(response.getETag().contains("\"")); // Quotes should be scrubbed from etag header values
+        assertNotNull(response.getLastModified());
+        assertNotNull(response.getRequestId());
+        assertNotNull(response.getVersion());
+        assertNotNull(response.getDateProperty());
+        assertTrue(response.getMetadata() == null || response.getMetadata().isEmpty());
+        assertEquals(BlobType.BLOCK_BLOB, response.getBlobType());
+        assertNull(response.getCopyCompletionTime()); // tested in "copy"
+        assertNull(response.getCopyStatusDescription()); // only returned when the service has errors; cannot validate.
+        assertNull(response.getCopyId()); // tested in "abort copy"
+        assertNull(response.getCopyProgress()); // tested in "copy"
+        assertNull(response.getCopySource()); // tested in "copy"
+        assertNull(response.getCopyStatus()); // tested in "copy"
+        assertTrue(response.isIncrementalCopy() == null || !response.isIncrementalCopy()); // tested in PageBlob."start incremental copy"
+        assertNull(response.getDestinationSnapshot()); // tested in PageBlob."start incremental copy"
+        assertNull(response.getLeaseDuration()); // tested in "acquire lease"
+        assertEquals(LeaseStateType.AVAILABLE, response.getLeaseState());
+        assertEquals(LeaseStatusType.UNLOCKED, response.getLeaseStatus());
+        assertTrue(response.getContentLength() >= 0);
+        assertNotNull(response.getContentType());
+        assertNull(response.getContentMD5());
+        assertNull(response.getContentEncoding()); // tested in "set HTTP headers"
+        assertNull(response.getContentDisposition()); // tested in "set HTTP headers"
+        assertNull(response.getContentLanguage()); // tested in "set HTTP headers"
+        assertNull(response.getCacheControl()); // tested in "set HTTP headers"
+        assertNull(response.getBlobSequenceNumber()); // tested in PageBlob."create sequence number"
+        assertEquals("bytes", response.getAcceptRanges());
+        assertNull(response.getBlobCommittedBlockCount()); // tested in AppendBlob."append block"
+        assertTrue(response.isServerEncrypted());
+        assertEquals(AccessTier.HOT.toString(), response.getAccessTier());
+        assertTrue(response.isAccessTierInferred());
+        assertNull(response.getArchiveStatus());
+        assertNotNull(response.getCreationTime());
+        // Tag Count not in BlobProperties.
```

Comment:
Are these not present because we need to add them?

Reply:
Yeah we can add them when we upgrade to 73